tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -217,11 +216,6 @@
         u'/docker/api/v2/containers/:id/logs',
         u'/docker/api/v2/containers/:id/power',
     ),
-    u'content_reports':(
-        u'/katello/api/content_reports/status_trend',
-        u'/katello/api/content_reports/system_status',
-        u'/katello/api/content_reports/system_trend',
-    ),
```

Comment:
Content-reports URL's are related to gutterball feature, which is going to be completely removed from Sat6.2. So I think that removal is valid.